components/cice/cime_config/buildlib

```diff
@@ -42,19 +42,33 @@ if (-f "Filepath") {
 #  force gmake to recompile if any of the cppdefs are different 
 #-------------------------------------------------------------- 
 my $recompile = 'FALSE';
-if (-e "$OBJROOT/ice/obj/CCSM_cppdefs") {
+my $sysmod = "cp -f $OBJROOT/ice/obj/CIME_cppdefs.new $OBJROOT/ice/obj/CIME_cppdefs";
```

Comment:
There is probably a way to do this without duplicating so much code. Just store the correct cppdef filename root in a variable.

Reply:
Ok  I changed cice/cime_config/buildlib/ to store
the correct cppdef filename root in a variable
which increases code efficiency and reduces
excess duplication of code.